Upload Part S3 operation examples about correct way yo use (#2340)
mulesoft/docs-connectors: amazon-s3/6.2/modules/ROOT/pages/amazon-s3-connector-reference.adoc

[NOTE]
After you initiate a multipart upload and upload one or more parts, you must either complete or abort the multipart upload to stop getting charged for storing the uploaded parts. After you complete or abort the multipart upload, Amazon S3 frees up the parts storage and stops charging you for it.

==== Common Cases for the Operation

> * When you upload large file content in parts (from, for example, a FTP Connector *Read* operation or SFTP Connector *On New or Updated File* source of a Mule app), you must configure the *non repeatable stream* strategy in these operations. Additionally, in the *Upload part* operation specify in the *Content length* field the number of bytes to consume from the content.

Reviewer: ```suggestion
* When you upload content of a large file in parts (for example, an FTP Connector *Read* operation or SFTP Connector *On New or Updated File* source of a Mule app), you must configure the *non repeatable stream* strategy in these operations. You must also specify the number of bytes to consume from the content in the *Upload part > Content length* field.
```

> * When you upload many small file contents, you can either configure *repeatable stream* or *non repeatable stream* strategy in the FTP or SFTP Connector operations. In the *Upload part* operation, you can either specify or not in the *Content length* field the entire file size you want to upload.

Reviewer: ```suggestion
* When you upload the content of many small files, you can configure either the *repeatable stream* or *non repeatable stream* strategy in the FTP or SFTP Connector operations. In the *Upload part* operation, you can optionally specify the entire file size in the *Content length* field.
```
